microsetta_public_api/config.py

```diff
@@ -1,6 +1,23 @@
 import os
+import json
 from microsetta_public_api.exceptions import ConfigurationError
 
+# NOTE: importlib replaces setuptools' pkg_resources as of Python 3.7
+# See: https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package # noqa
+
+PACKAGE_NAME = __name__.split('.')[0]
+CONFIG_FILE = "server_config.json"
```

Comment:
Any way to make this less hard-coded? E.g.,

```suggestion
CONFIG_FILE = os.getenv("MPUBAPI_CFG", "server_config.json")
```

Reply:
Would have some implications for the `importlib.resources`/`pkg_resources` code below.

Reply:
I think we'd want to tackle that consistently across the projects. The other three are using `server_config.json` relative to the package

Reply:
I'm good with aiming for consistency across projects. But I would also love to have a way to maintain multiple configs and not have to keep making changes the same "server_config.json". It's so much easier to just change a flag or environment variable.

Reply:
And it isn't impossible to have the same default behavior as the other projects while opening this project for extension.

Reply:
I mean, `cp some_test_config.json server_config.json` is about the same effort as `export SOMEARGUMENT=some_test_config.json`. I don't have a strong opinion though on how this evolves as long as it is easy to use with supervisord/gunicorn, and is consistent across the microsetta services.